flutter_local_notifications_windows/lib/src/details/notification_parts.dart

```diff
@@ -29,24 +32,67 @@ enum WindowsImageCrop {
 }
 
 /// An image in a Windows notification.
+///
+/// Windows supports a few different URI types, and supports them differently
+/// depending on if your app is packaged as an MSIX. Refer to the following:
+///
+/// | URI | Debug | Release (EXE) | Release (MSIX) |
+/// |--------|--------|--------|--------|
+/// | `http(s)://` | ❌ | ❌ | ✅ |
+/// | `ms-appx://` | ❌ | ❌ | ✅ |
+/// | `file:///`   | ✅ | ✅ | 🟨 |
+/// | `assetUri()` | ✅ | ✅ | ✅ |
+///
+/// Each URI type has different uses:
+/// - For Flutter assets, use [assetUri], which return the correct file URI
+/// for debug and release (exe) builds, and an `ms-appx` URI in MSIX builds.
+/// - For images from the web, use an `https` or `http` URI, but note that
+/// these only work in MSIX apps. If you need a network image without using
+/// MSIX, consider downloading it directly and using a file URI after. Also
+/// note that showing the notification will cause the image to be downloaded,
+/// which could cause a small delay. Try to use small images.
+/// - For images that come from the user's device, or have to be retrieved at
+/// runtime, use a file URI, but as always, be aware of how paths might change
+/// from your device to your users. Note that file URIs must be absolute
+/// paths, not relative, which can be complicated if referring to MSIX assets.
+/// - For images that are bundled with your app but not through Flutter, use
+/// an `ms-appx` URI.
 class WindowsImage extends WindowsNotificationPart {
-  /// Creates a Windows notification image.
-  const WindowsImage.file(
-    this.file, {
+  /// Creates a Windows notification image from an image URI.
+  const WindowsImage(
+    this.uri, {
     required this.altText,
     this.addQueryParams = false,
     this.placement,
     this.crop,
   });
 
+  /// Creates a URI for a [Flutter asset](https://docs.flutter.dev/ui/assets/assets-and-images#loading-images).
+  ///
+  /// - In debug mode, resolves to a file URI to the asset itself
+  /// - In non-MSIX release builds, resolves to a file URI to the bundled asset
+  /// - In MSIX releases, resolves to an `ms-appx` URI from [Msix.assetUri].
+  static Uri assetUri(String assetName) {
```

Comment:
As this isn't a constructor but a method, can you rename it so it reads like one to convey that an action is being taken? I was originally thinking `resolve` should be part of it but saw [this](https://api.flutter.dev/flutter/dart-ui_web/AssetManager/getAssetUrl.html) exists that `getAssetUri()` could potentially work too. If you use the latter though then would be appropriate to update the API docs e.g. replace "creates" with "returns".

Similar comment applies to the method in `MsixUtils`

Reply:
Done